hbase_sql/src/test/java/com/splicemachine/derby/utils/SqlshellIT.java

```diff
@@ -484,4 +484,25 @@ public void testLexerErrorMessage() {
                 "5:\tL_QUANTITY DECIMAL(15,2),\n" +
                 ".\n");
     }
+
+    @Test
+    public void testDisplayTriggerInfo() {
+        // Display trigger info should be by default off
+        execute("SELECT * FROM ABC;\n",
+                "DEF        \n" +
+                        "-----------\n" +
+                        "\n" +
+                        "0 rows selected\n");
+        // set display trigger info on and display it.
+        execute("displaytriggerinfo on;\n", "");
+        execute("SELECT * FROM ABC;\n",
+                "DEF        \n" +
+                        "-----------\n" +
+                        "\n" +
+                        "0 rows selected\n" +
+                        "triggerName                                                                                                                     |triggerId                           |txnId               |queryId                             |parentQueryId                       |timeSpent           |numRowsModified     \n" +
+                        "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n" +
+                        "\n" +
+                        "0 rows selected\n");
```

Comment:
this needs a
```
try { 
    execute("displaytriggerinfo on;\n", "");
    ...
} finally {
    execute("displaytriggerinfo off;\n", "");
}
```
otherwise tests after that one will have the triggerinfo still on and return different results.